### PR TITLE
[Cleanup] Clients | Products | Invoices -> Permissions Adjustment && Additional Tests

### PR DESCRIPTION
**Pull request by Civolilah**
@beganovich @turbo124 The PR involves significant cleanup related to permissions. This cleanup spans the main navigation bar, the dashboard, the dropdown where users switch companies, the cleanup in the quick creation popup, and the primary focus of the cleanup is on the `Clients`, `Products`, and `Invoices` pages, along with everything related to them. Let me know your thoughts.

### CODE REVIEW

**Review by beganovich — APPROVED** [review]
Looks good, but I can't seem to get tests to pass. 

Is it working for you guys? @turbo124 @Civolilah. Also, it might be the correct time to invest and make them work on CI too.

Other than that, looks good to me.

**Comment by Civolilah** [comment]
> Looks good, but I can't seem to get tests to pass.
> 
> Is it working for you guys? @turbo124 @Civolilah. Also, it might be the correct time to invest and make them work on CI too.
> 
> Other than that, looks good to me.

@beganovich @turbo124 I guess you ran all the tests together. Actually, since this PR is related to main permission aspects and elements associated with `Dashboard`, `Clients`, `Products`, and `Invoices`, the tests that should work when you run them are in:

- clients.spec.ts
- dashboard.spec.ts
- invoices.spec.ts
- products.spec.ts
- navigation.spec.ts

So, the PR is, however, very large because there are a lot of changes. If we include all the other tests, it will be very hard to review. The changes made in other hooks related to returning actions, such as `Expenses`, were necessary because the Expenses table is on the `Client` show page. To ensure the Client show page works perfectly fine, I needed to adjust actions on Expenses, for example. However, following PRs related to permissions will be smaller due to this fact.

In the upcoming PRs related to permissions, I will adjust the functionality along with tests. By the end of this task related to permissions, almost all files should work perfectly fine when you run tests. If there are any additional files where the tests are not fixed, in the last PR related to permissions, I will also adjust them. So, we can consider running tests on CI as expected.

Therefore, if you run tests locally on the mentioned files, all tests should pass.

Let me know your thoughts.

**Comment by beganovich** [comment]
Hey @Civolilah, I think it's a good time to make all tests green. Let's aim for that in this PR!

**Comment by Civolilah** [comment]
> Hey @Civolilah, I think it's a good time to make all tests green. Let's aim for that in this PR!

@beganovich Sure! Let me fix tests on this one.

**Comment by beganovich** [comment]
Ping @Civolilah for merge conflicts.

**Comment by Civolilah** [comment]
> Ping @Civolilah for merge conflicts.

@beganovich Sure, done!

File: playwright.config.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ config();
 export default defineConfig({
   testDir: './tests/e2e',
   /* Maximum time one test can run for. */
-  timeout: 10 * 1000,
+  timeout: 10 * 1500,
```
Comment (Civolilah) [inline playwright.config.ts:16]: @beganovich Some tests for invoices take longer than 10 seconds, which is the reason for increasing this timeout.

**Pull request merged**